From 2a479acd235283835a2fc82f9a439963bc4497e5 Mon Sep 17 00:00:00 2001
From: Marco Chiotti <marcox.chiotti@intel.com>
Date: Thu, 31 Mar 2016 14:43:08 +0200
Subject: [PATCH 2/6] [BT] Remove wrong call to set_wake_locked()

bt_lpm_rfkill_set_power() is called before
bt_lpm.tty_dev is initialized so the calls to
set_wake_locked() shall be removed from it

Change-Id: I1d5ab82c60a97de8a0dae796788fb831580114f8
Signed-off-by: Andrew Wyper <andrewx.wyper@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-28237
Reviewed-on: https://android.intel.com/#/c/491954/
Reviewed-on: https://android.intel.com:443/491954
---
 drivers/misc/bt_lpm/bt_lpm.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/misc/bt_lpm/bt_lpm.c b/drivers/misc/bt_lpm/bt_lpm.c
index f70d003..30a0083 100644
--- a/drivers/misc/bt_lpm/bt_lpm.c
+++ b/drivers/misc/bt_lpm/bt_lpm.c
@@ -189,7 +189,6 @@ static int bt_lpm_rfkill_set_power(void *data, bool blocked)
 		gpio_set_value(bt_lpm.gpio_reset, 1);
 #else
 		gpio_set_value(bt_lpm.gpio_enable_bt, 1);
-		set_wake_locked(1);
 
 #endif
 		pr_debug("%s: turn BT on\n", __func__);
@@ -199,7 +198,6 @@ static int bt_lpm_rfkill_set_power(void *data, bool blocked)
 		gpio_set_value(bt_lpm.gpio_reset, 0);
 #else
 		gpio_set_value(bt_lpm.gpio_enable_bt, 0);
-		set_wake_locked(0);
 #endif
 		pr_debug("%s: turn BT off\n", __func__);
 	}
-- 
1.9.1

